bronze.py

```diff
@@ -0,0 +1,45 @@
+from flask_dance.consumer import OAuth2ConsumerBlueprint
+from functools import partial
+from flask.globals import LocalProxy, _lookup_app_object
+
+try:
+    from flask import _app_ctx_stack as stack
+except ImportError:
+    from flask import _request_ctx_stack as stack
+
+def make_bronze_blueprint(
```

Comment:
add new parameter (in first position) "test", if true use the "test.bronze.exchange" if false use "bronze.exchange"

Reply:
won't it be better to just change it like this:
```
def make_bronze_blueprint(
    scope,
    redirect_url=None,
    redirect_to=None,
    login_url=None,
    authorized_url=None,
    session_class=None,
    storage=None,
):
    bronze_bp = OAuth2ConsumerBlueprint(
        "bronze",
        __name__,
        client_id=app.config["CLIENT_ID"],
        client_secret=app.config["CLIENT_SECRET"],
        scope=scope,
        base_url=app.config["BRONZE_ADDRESS"]+"/oauth/v1/",
        authorization_url=app.config["BRONZE_ADDRESS"]+"/oauth/v1/Auth",
        token_url=app.config["BRONZE_ADDRESS"]+"/oauth/v1/Token",
        token_url_params={"include_client_id": True},
        redirect_url=redirect_url,
        redirect_to=redirect_to,
        login_url=login_url,
        authorized_url=authorized_url,
        session_class=session_class,
        storage=storage,
    )
```


since its part of the config already, I'm just using whats already there.